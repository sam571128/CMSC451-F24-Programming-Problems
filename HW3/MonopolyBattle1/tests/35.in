55 3025 1000
23 22
25 11
1 49
32 50
14 4
8 51
35 8
44 13
43 27
27 31
52 37
55 3
35 7
25 33
26 4
5 13
13 20
11 6
48 39
15 44
15 8
25 30
32 10
9 12
17 39
35 54
25 21
26 22
13 25
45 14
7 8
52 51
19 37
19 4
39 54
8 33
40 42
17 15
35 31
16 55
52 44
45 36
5 12
19 45
6 43
26 8
16 36
32 24
8 53
24 7
43 44
4 48
13 30
51 44
6 25
16 26
21 22
30 31
27 10
5 20
15 4
20 53
38 29
40 14
47 53
48 2
36 32
51 16
12 42
40 53
9 39
33 19
23 34
46 27
5 18
38 36
30 2
38 53
27 13
24 9
16 33
14 54
46 22
31 3
40 29
16 41
16 10
50 9
3 54
31 15
45 22
9 43
5 1
33 12
49 23
28 3
12 23
4 42
23 39
52 45
3 4
11 5
55 36
39 52
48 23
34 51
24 41
19 27
23 12
15 46
26 50
31 24
47 54
45 43
44 22
50 43
52 47
11 37
48 45
23 54
21 51
15 22
15 6
30 45
31 28
19 34
4 8
38 12
11 26
26 12
38 52
30 23
4 55
9 20
15 10
39 55
19 15
44 55
39 8
35 4
27 43
34 9
49 12
24 18
41 10
1 16
14 36
48 49
46 48
3 31
54 49
50 32
51 51
23 45
36 31
40 30
51 29
28 8
49 14
16 54
23 43
2 38
48 1
48 29
42 55
46 13
47 21
13 6
20 9
38 48
3 45
45 17
30 22
46 45
35 32
51 35
49 42
51 54
21 18
28 14
33 26
50 51
49 1
34 21
19 2
3 19
14 34
38 1
49 19
55 6
12 5
55 2
14 14
40 39
22 1
8 35
35 20
19 26
24 22
21 49
42 7
18 3
14 32
38 17
18 18
35 48
24 54
6 34
53 33
37 15
50 17
2 35
24 26
35 50
28 31
15 28
26 43
55 10
43 25
11 46
13 36
12 46
39 4
55 8
44 54
21 34
50 55
36 1
6 52
3 11
54 4
41 36
12 15
4 6
32 16
29 10
35 19
16 35
20 12
45 24
36 52
3 43
28 6
20 1
14 15
6 4
21 11
4 41
53 23
3 35
19 44
32 12
55 47
38 49
34 32
13 52
33 33
10 36
51 34
4 16
8 26
29 25
20 52
1 44
8 41
46 55
15 2
31 49
40 24
2 39
21 52
48 33
45 13
49 41
52 26
21 26
35 33
15 40
37 44
30 52
19 46
38 45
48 19
36 50
21 36
7 44
33 28
40 18
36 28
54 42
54 34
19 10
36 5
46 26
18 27
4 4
27 20
47 16
19 21
22 9
50 40
2 2
18 55
31 44
23 15
39 24
40 43
15 11
48 30
3 49
16 43
20 37
2 19
24 48
27 3
36 21
37 22
38 54
31 16
45 33
13 22
16 17
28 51
18 44
38 22
37 38
18 16
51 46
18 12
46 3
35 47
53 47
17 28
42 17
54 19
22 6
10 2
42 8
34 11
48 10
2 48
41 23
45 12
48 15
31 25
40 36
37 42
7 10
38 26
18 49
53 55
29 53
26 47
16 31
31 52
11 12
23 41
22 50
6 7
50 8
52 2
53 24
11 1
3 44
14 51
33 47
35 37
14 48
28 30
7 48
39 51
54 25
6 17
24 28
48 12
52 42
15 20
31 47
50 28
42 45
28 2
35 38
27 25
49 36
43 29
1 2
3 23
38 16
40 50
20 14
10 45
35 17
32 1
26 24
53 1
12 47
8 54
4 34
2 1
34 20
55 38
3 34
43 42
21 12
35 9
42 46
37 32
48 24
20 19
7 22
4 36
25 4
9 32
24 40
25 41
2 42
39 25
11 22
37 36
5 40
41 44
32 40
32 27
41 9
1 45
44 43
11 36
37 1
30 44
44 12
32 2
2 14
4 40
49 51
44 52
38 41
32 37
52 18
3 28
31 50
44 50
1 1
23 9
22 39
47 12
5 54
51 31
4 46
29 16
7 4
29 30
17 21
35 16
20 11
6 20
50 4
45 41
38 42
32 25
15 19
1 25
17 47
15 29
8 50
45 28
14 5
44 30
6 54
7 1
11 25
12 44
11 24
30 8
17 36
7 13
27 41
25 55
40 3
44 32
50 14
40 10
20 23
50 33
28 47
8 55
13 17
31 6
23 28
27 35
16 13
21 47
34 23
2 21
16 8
36 55
47 9
40 2
10 42
18 2
37 34
37 29
1 48
21 23
11 16
5 15
25 25
47 8
33 35
21 30
29 54
48 26
51 18
49 4
55 22
54 18
39 37
22 36
10 17
5 50
24 44
23 31
28 46
37 3
4 18
41 5
50 54
14 43
1 50
32 19
14 41
37 7
33 21
36 22
51 37
52 35
47 19
12 28
43 3
40 38
17 24
9 35
22 17
37 23
37 31
30 13
30 43
6 35
23 10
43 40
55 27
8 31
55 37
36 14
52 48
2 44
4 28
19 6
12 27
34 10
55 43
21 14
7 24
2 54
24 21
15 35
50 39
11 4
30 6
45 19
55 33
18 53
50 48
10 28
47 55
3 6
37 6
22 12
10 25
22 32
42 22
40 44
24 34
2 7
31 48
31 32
26 34
37 14
34 34
8 11
24 45
2 3
13 12
16 7
53 9
33 44
28 21
1 29
9 23
29 26
23 47
38 9
4 32
40 7
50 31
44 7
3 37
29 50
45 45
3 2
55 51
51 48
40 23
10 32
39 26
46 31
36 30
46 12
12 39
53 38
28 12
26 5
19 12
14 22
4 22
33 24
40 26
27 53
33 3
32 36
37 48
16 21
13 49
46 24
34 18
52 31
42 32
55 45
41 6
38 38
28 20
24 46
47 1
54 16
29 31
20 36
16 4
34 43
39 7
16 11
48 14
31 5
18 15
14 12
10 55
35 13
22 27
46 9
9 30
46 21
17 54
38 31
39 30
15 38
24 49
21 27
51 22
11 8
13 24
44 44
16 42
53 19
44 45
8 40
34 4
2 53
1 33
52 16
40 41
12 14
36 44
12 4
8 21
45 16
39 10
32 29
51 23
12 7
7 49
46 20
37 52
15 1
15 3
3 24
8 15
52 6
6 1
4 12
34 5
1 18
8 52
25 46
23 5
19 7
40 46
5 8
1 17
54 27
15 23
45 18
5 21
32 18
45 52
10 30
25 47
54 55
15 13
34 45
55 29
19 23
19 42
11 30
39 45
23 53
26 25
55 24
15 21
15 55
10 49
30 16
52 13
22 4
55 30
25 53
43 49
26 38
9 47
21 17
47 43
21 25
10 44
22 21
52 23
20 25
28 17
47 29
49 47
50 29
37 39
32 14
13 51
43 2
55 23
11 14
39 1
34 35
27 52
46 8
40 54
41 53
20 8
17 49
51 42
31 33
18 36
51 49
20 54
38 13
1 23
25 43
9 28
27 54
46 53
27 6
21 20
22 18
36 45
20 44
51 14
48 22
25 35
41 7
7 9
31 26
38 25
13 38
5 17
3 12
43 48
3 18
43 15
22 25
34 19
53 39
51 55
26 15
35 43
1 22
55 26
44 11
19 47
45 49
49 52
52 30
27 22
46 33
22 37
6 16
42 11
11 2
21 32
3 15
3 52
5 36
49 16
16 24
48 48
23 4
32 46
18 4
23 42
26 40
54 3
54 6
39 11
17 3
36 40
31 51
15 45
50 16
47 47
45 40
24 6
10 35
14 40
34 15
10 18
22 20
52 11
23 14
18 31
14 44
21 2
30 38
49 50
17 55
27 50
45 53
51 41
2 16
34 48
19 39
9 48
45 51
19 50
35 5
55 15
28 4
21 15
2 34
5 4
11 44
52 3
55 4
55 14
28 55
42 5
26 36
29 32
30 1
35 36
6 46
23 8
43 22
50 27
9 40
50 49
55 53
54 43
53 3
18 30
44 40
4 11
55 12
42 33
48 28
23 46
9 1
16 9
24 37
32 13
54 22
8 6
33 22
26 28
17 41
47 14
30 37
13 46
51 39
21 9
4 21
11 11
18 37
27 48
3 5
28 34
43 1
19 30
33 14
3 36
9 3
20 13
41 31
50 35
17 27
31 14
55 39
12 21
10 34
11 28
53 26
54 46
45 35
49 38
1 53
3 32
28 10
10 52
48 54
23 40
27 19
17 48
25 37
6 42
38 3
54 33
6 8
55 50
47 39
16 50
12 24
42 50
48 13
14 52
40 12
7 30
39 47
16 47
11 7
51 50
6 38
8 38
37 54
17 53
40 8
10 11
20 2
8 7
54 52
24 13
55 9
9 10
16 22
48 17
6 23
1 52
12 50
27 38
43 5
28 19
18 6
39 42
42 44
43 13
17 7
47 15
12 53
4 39
52 27
50 1
39 29
19 1
41 16
39 35
17 40
9 11
45 11
53 21
32 44
54 15
23 11
39 46
54 38
12 20
55 52
34 2
17 26
12 38
28 1
28 43
55 11
14 37
38 7
6 2
29 52
21 31
54 11
32 38
39 49
4 37
30 33
45 29
32 11
3 47
14 19
2 36
6 37
28 27
16 12
25 50
26 45
6 26
41 24
30 15
14 7
36 39
27 34
16 23
27 33
16 48
31 27
2 18
52 22
3 50
32 43
20 55
33 11
29 5
17 35
4 33
42 18
46 4
26 11
36 18
10 47
34 28
48 44
50 53
21 35
19 5
30 47
54 51
23 38
30 30
1 32
30 28
8 49
4 17
41 12
25 39
46 25
29 43
34 33
40 17
45 25
26 20
24 11
9 22
50 50
7 55
32 48
52 15
49 29
12 8
16 16
10 38
41 48
12 22
42 28
7 46
54 36
8 45
55 42
50 15
27 21
55 28
22 22
29 45
37 35
53 45
8 25
20 15
7 14
53 48
38 43
37 16
19 17
38 20
14 29
1 27
4 43
38 32
42 40
9 17
42 29
49 39
2 25
36 33
25 34
27 11
6 3
13 35
18 19
14 11
3 20
50 41
51 2
16 37
40 6
19 55
41 21
40 5
18 7
51 32
22 23
30 24
24 50
55 1
54 1
26 41
17 8
51 52
24 20
25 9
40 4
13 13
32 54
12 36
41 40
44 29
20 40
32 4
47 52
49 2
7 20
47 40
22 16
22 38
45 4
5 28
10 19
54 29
20 10
11 53
45 31
16 14
45 55
47 2
46 32
52 21
32 15
39 12
4 35
48 35
3 53
18 11
30 49
30 4
18 23
10 53
2 40
24 47
53 8
14 1
35 44
19 43
22 44
4 49
39 31
50 6
19 9
45 1
18 9
44 15
50 2
32 55
29 40
18 40
53 46
35 11
46 54
48 32
37 43
38 15
46 2
21 1
54 28
53 29
41 46
5 48
27 32
44 31
16 3
1 13
20 47
4 7
10 16
8 27
33 37
25 3
22 43
34 54
32 32
9 34
4 38
1 4
24 24
13 32
5 42
53 41
8 32
55 46
42 49
54 8
46 42
17 42
33 40
53 13
25 14
14 8
4 20
36 4
49 6
11 50
45 47
47 17
42 26
18 48
53 12
36 53
21 4
7 41
9 6
22 15
47 33
14 25
53 6
23 36
2 37
15 50
52 4
35 52
9 14
33 46
52 36
54 47
51 5
13 1
19 33
5 30
20 39
19 16
33 32
22 13
47 30
6 18
36 11
37 25
49 25
37 45
53 43
53 37
10 3
7 17
44 21
46 23
50 13
45 48
26 42
10 24
14 49
20 26
39 2
1 5
24 31
8 30
52 24
41 37
5 55
21 19
44 48
39 14
27 44
33 9
52 41
3 55
4 13
39 22
22 55
33 1
4 53
52 33
52 28
26 26
4 26
17 13
27 14
18 13
9 16
25 36
19 8
13 50
42 15
17 18
50 47
16 6
43 36
18 29
32 28
50 37
20 51
20 30
35 41
46 11
5 34
50 34
3 42
18 38
48 20
53 20
5 53
1 7
42 34
3 46
30 17
28 52
43 55
12 41
21 37
27 40
20 22
30 3
34 40
18 35
5 31
30 9
29 55
18 54
41 17
41 14
48 16
21 40
15 27
4 44
23 33
48 21
47 7
18 5
17 6
34 3
1 19
2 26
13 26
36 7
10 22
51 38
39 17
52 12
40 37
50 11
37 20
22 34
55 31
44 1
55 20
40 48
13 40
35 15
40 51
37 49
27 45
4 52
27 15
38 2
49 40
49 21
28 33
12 9
33 4
15 43
48 27
5 14
50 26
8 4
50 5
39 20
21 28
40 1
30 7
9 52
8 13
14 17
3 48
26 53
18 45
48 5
46 6
6 53
25 10
8 34
44 23
21 21
7 31
24 27
31 17
28 53
44 10
13 16
45 37
37 11
20 41
27 42
44 17
17 44
54 17
10 7
37 8
19 41
26 46
21 41
18 41
31 36
23 17
6 50
19 40
7 25
27 17
1 9
33 25
10 9
17 12
36 25
44 34
41 11
18 46
12 18
39 50
22 2
1 20
8 8
10 54
6 24
14 39
18 51
2 29
53 36
50 38
22 49
1 24
43 43
23 55
42 38
3 27
38 21
20 46
53 4
19 38
51 10
43 18
30 25
31 11
23 48
51 4
3 8
14 53
51 43
8 44
42 48
30 40
30 36
3 39
39 9
44 16
43 33
50 42
50 7
23 32
13 42
46 47
54 14
50 52
11 54
12 40
13 48
36 42
22 28
43 37
39 19
55 16
38 37
55 21
10 46
47 45
26 27
38 33
55 44
1 35
7 6
32 41
33 55
32 8
46 1
37 12
32 42
35 45
16 53
55 7
32 31
28 32
49 26
12 51
3 3
54 50
13 4
19 18
17 17
21 10
15 31
30 54
29 46
2 52
39 15
5 19
20 32
10 29
20 24
38 27
47 18
46 38
15 48
27 27
25 18
33 31
7 39
38 23
51 7
51 47
52 55
47 6
46 16
26 3
13 28
47 35
23 26
6 55
51 19
34 44
41 49
27 18
35 3
36 51
52 1
15 9
52 14
13 37
19 25
27 30
46 10
5 9
35 25
53 52
47 49
29 39
42 52
9 4
41 18
26 32
9 7
31 30
33 49
25 32
53 16
46 37
10 23
39 27
1 31
11 31
43 28
42 47
14 33
35 27
31 18
22 8
1 15
34 26
46 39
23 44
49 35
50 45
31 22
51 11
43 34
13 23
34 25
12 49
29 17
43 52
4 10
21 38
7 18
6 5
21 39
21 53
1 8
1 14
9 25
22 26
37 19
46 51
32 20
23 6
20 18
2 8
49 43
9 54
1 47
14 13
14 45
33 34
12 48
19 19
26 14
33 42
44 18
31 4
2 32
42 53
24 35
47 46
25 29
53 42
29 44
41 19
1 51
39 38
15 14
27 24
2 45
17 45
4 15
47 28
6 6
2 17
7 21
6 28
41 1
47 26
36 12
36 10
33 50
27 36
25 15
11 21
8 37
46 5
13 15
48 11
29 3
43 51
1 37
13 39
12 16
14 21
4 14
40 49
19 29
20 4
26 37
1 54
13 14
23 52
37 18
44 9
51 9
14 26
16 45
23 50
2 27
25 49
8 39
39 36
23 27
31 9
30 51
46 43
1 46
13 5
34 13
41 42
27 4
8 42
15 34
15 52
20 20
34 31
5 2
6 10
47 31
13 8
49 32
17 46
19 49
19 48
44 8
43 35
45 26
2 13
40 15
7 50
33 17
15 16
53 17
12 12
6 13
52 5
33 39
28 7
24 12
41 52
47 50
15 12
49 9
14 50
43 10
24 36
15 30
25 8
12 2
22 53
28 25
28 38
29 23
44 28
9 24
9 31
8 19
43 31
38 39
12 26
3 21
21 5
5 6
48 4
53 53
35 24
43 38
52 32
16 44
8 48
19 31
36 49
47 42
7 19
10 50
20 45
5 41
15 53
16 40
44 49
34 38
25 17
29 22
18 42
3 13
9 44
7 7
9 45
38 40
44 36
29 42
24 52
20 35
48 9
54 48
29 20
5 43
53 44
51 27
29 27
32 3
36 27
6 39
24 33
45 42
52 25
5 46
14 47
43 12
25 31
8 24
27 12
31 2
26 7
26 55
35 18
30 29
36 8
48 34
47 32
1 43
6 11
18 10
11 42
14 2
40 45
10 10
12 19
18 34
4 29
28 26
17 11
29 12
5 22
41 8
38 28
6 12
15 54
18 25
41 43
13 45
14 55
7 37
29 49
50 23
19 3
43 41
1 3
6 44
24 16
34 17
34 41
39 34
44 20
33 10
13 47
55 17
7 11
13 34
2 49
2 41
38 30
17 52
30 34
26 19
34 27
2 11
23 20
43 19
2 51
10 33
26 31
14 10
48 25
18 20
29 48
9 8
53 34
23 37
31 38
36 15
1 38
33 13
31 20
19 51
8 43
45 9
35 42
45 20
51 33
31 10
11 47
53 2
38 10
49 30
18 47
45 10
47 13
20 33
38 55
47 51
5 32
20 49
4 27
27 5
11 41
53 35
53 14
49 34
28 24
37 51
16 30
18 43
2 31
11 52
41 47
52 17
32 34
42 39
28 35
23 29
33 51
47 38
10 8
49 24
28 23
21 6
34 52
50 46
32 30
36 17
37 53
41 2
37 5
46 19
3 33
41 28
37 17
9 42
10 6
11 29
21 55
9 26
40 28
26 10
51 53
28 37
25 42
10 26
46 41
53 15
7 12
40 13
45 2
39 21
16 2
22 52
31 55
28 42
24 43
9 53
28 13
29 41
40 40
40 33
19 53
38 46
24 10
41 27
10 5
12 1
31 21
26 9
8 14
37 24
5 3
41 20
41 41
45 39
23 35
39 43
36 24
25 22
35 28
36 16
33 15
15 41
25 44
55 19
49 54
49 11
34 39
21 46
24 17
21 8
30 21
30 41
21 44
9 29
54 23
3 17
37 50
36 20
25 20
24 39
34 24
52 29
35 40
35 14
35 55
47 41
55 55
3 30
36 46
34 14
20 34
54 24
32 53
35 10
46 14
22 3
14 46
16 39
54 53
5 49
13 55
22 19
8 28
44 24
13 21
10 12
43 32
15 37
32 9
55 32
43 47
38 47
45 44
34 49
10 39
37 10
41 35
16 38
24 8
47 5
40 55
27 1
17 4
23 1
28 54
5 52
28 15
50 10
10 40
25 6
36 13
4 2
18 50
11 49
44 6
2 46
26 54
17 30
40 19
13 41
6 30
15 18
7 5
19 52
39 33
46 29
22 24
15 5
24 19
52 39
52 8
2 5
6 41
33 45
5 5
39 6
26 49
46 44
50 22
52 9
51 30
19 14
2 20
44 26
1 34
26 23
7 34
52 52
18 52
44 42
47 36
14 30
51 8
48 51
22 42
31 7
9 41
18 21
29 28
49 46
7 2
17 50
36 47
37 30
25 16
48 38
22 29
3 10
28 16
13 9
48 37
31 54
12 30
44 14
42 20
51 13
16 27
43 53
37 47
5 11
49 31
28 50
27 37
28 44
49 45
5 10
16 34
35 46
55 13
54 41
23 18
11 9
24 1
50 30
31 8
39 28
46 40
30 11
47 37
25 26
3 29
31 34
28 48
6 9
10 15
40 21
6 14
29 38
14 20
6 19
42 21
15 47
20 16
37 26
49 17
45 6
25 19
23 19
24 32
22 48
22 35
32 22
27 46
52 40
54 10
30 5
22 10
42 36
49 10
18 24
48 53
43 14
28 9
51 3
31 45
42 6
44 39
49 49
47 11
36 9
4 19
8 36
28 22
33 41
41 38
26 6
36 23
5 35
19 35
7 54
21 24
51 36
19 54
39 41
16 1
45 23
10 41
38 4
3 14
48 50
25 7
49 15
8 47
35 21
16 18
43 7
55 25
34 36
19 22
6 47
36 6
50 21
22 46
22 5
4 25
5 7
41 50
25 40
16 28
30 26
53 32
37 46
48 18
54 32
20 31
26 35
43 45
17 19
22 7
43 24
29 29
54 30
3 51
41 34
4 5
21 13
48 40
52 46
44 2
1 11
13 3
31 46
35 34
44 19
50 25
42 14
36 54
24 3
2 28
35 2
6 15
3 1
10 37
55 48
54 13
11 27
13 33
26 29
35 53
54 21
55 18
44 46
28 18
1 28
41 51
12 6
42 9
47 22
32 49
54 45
28 11
10 43
31 35
12 34
3 16
7 40
27 9
51 40
21 43
46 15
1 55
43 54
46 49
4 23
26 16
41 22
29 13
33 8
14 27
54 12
10 20
29 14
18 8
53 28
34 50
8 10
55 5
50 18
20 28
42 23
11 20
7 26
12 25
34 55
34 8
30 39
8 20
27 47
12 55
16 49
9 2
31 23
37 40
17 5
12 35
10 31
51 25
17 34
28 28
38 44
43 39
14 9
34 46
2 24
27 29
22 45
17 16
17 20
49 44
20 6
48 46
18 14
4 50
40 32
40 25
37 13
31 13
42 12
37 41
46 46
5 23
40 11
2 43
3 22
8 29
45 15
53 7
32 52
31 42
42 16
46 30
29 37
39 53
24 14
29 2
23 30
27 23
23 23
16 19
9 19
5 47
47 27
33 36
17 14
43 8
10 21
38 34
9 50
33 2
14 3
45 32
49 18
35 26
15 17
48 52
36 43
24 51
35 51
30 10
19 11
25 13
32 26
1 6
39 23
41 3
20 27
30 46
4 51
33 29
7 27
30 12
13 54
23 49
12 31
1 42
52 38
41 13
26 17
34 42
44 4
4 45
24 30
16 52
32 7
43 16
40 20
10 1
40 9
12 45
11 13
43 20
45 50
35 6
13 53
40 27
55 35
7 28
41 39
31 43
23 25
23 2
34 53
47 44
49 8
7 43
41 32
11 38
49 28
37 55
9 38
2 47
9 49
10 14
32 47
4 9
35 1
36 37
39 39
33 43
3 25
53 31
14 24
39 3
14 35
40 35
22 54
8 23
17 2
11 17
18 32
5 39
11 18
31 19
33 18
11 39
26 48
10 51
24 55
38 19
34 12
13 44
33 16
7 3
52 43
45 21
18 1
39 40
9 51
29 18
53 51
47 10
26 18
43 50
8 5
14 18
8 2
40 47
2 22
53 27
15 25
44 41
17 31
35 30
42 2
41 26
9 37
42 41
32 39
30 53
24 42
45 30
32 17
48 43
12 37
49 3
17 29
12 17
20 43
8 12
13 7
23 16
31 31
1 36
44 47
53 50
20 17
28 39
15 15
13 2
9 9
2 23
48 47
11 15
41 4
8 46
50 3
16 46
9 55
26 30
28 36
50 12
24 4
46 7
39 5
38 35
41 30
44 3
26 39
45 7
25 1
9 46
23 3
34 30
44 35
27 7
51 12
27 39
21 45
19 24
17 43
21 54
29 35
12 52
19 28
15 26
42 51
36 29
1 39
2 9
24 2
36 3
36 34
52 19
34 22
54 2
16 15
3 9
3 40
46 35
5 29
7 29
11 51
27 49
17 23
29 6
9 27
46 50
11 40
29 1
44 37
26 13
52 50
27 28
44 5
23 21
48 55
28 40
16 20
17 25
12 3
46 17
29 21
34 7
30 32
28 49
18 39
33 5
54 35
46 18
9 5
28 5
50 36
25 52
42 37
33 6
28 29
20 50
22 40
53 10
22 41
19 20
27 26
34 29
16 25
8 3
43 6
33 54
53 25
20 48
38 11
53 5
49 13
53 49
36 2
45 34
17 22
29 19
25 28
29 51
47 4
47 3
4 30
36 36
13 29
48 7
29 33
17 9
27 2
11 35
50 44
54 40
33 23
53 54
2 55
30 27
7 15
41 25
47 23
32 23
46 36
30 20
37 2
49 27
5 27
6 49
8 1
32 51
44 38
6 27
30 55
30 50
42 13
3 26
42 31
18 26
40 16
11 34
7 23
51 28
7 35
12 13
12 43
48 8
51 21
44 51
8 9
31 29
54 44
24 23
2 15
12 29
16 5
25 54
12 33
31 41
49 55
51 6
42 24
14 38
35 29
43 46
6 32
5 51
43 21
23 13
53 22
41 45
1 21
37 27
10 4
52 53
15 51
19 13
35 35
51 45
30 19
47 24
50 20
43 23
36 41
45 3
7 42
49 22
12 32
42 43
33 53
42 54
11 48
42 27
2 12
7 36
44 33
25 2
52 10
21 48
50 19
53 30
7 51
48 31
17 51
45 46
32 33
35 23
55 34
52 34
26 33
31 53
15 7
46 52
1 30
48 41
5 44
4 31
2 10
15 24
11 23
13 31
48 6
32 35
7 45
15 32
11 55
25 5
42 35
53 18
29 4
35 12
11 33
1 40
5 26
25 24
14 16
41 54
1 41
54 9
7 16
42 19
15 39
10 27
33 48
46 34
6 40
32 5
36 48
12 10
22 33
40 31
38 8
50 24
26 21
29 8
14 28
20 5
43 30
51 1
7 33
7 47
45 54
26 2
5 37
17 37
33 27
6 33
21 33
11 10
15 33
19 32
49 7
13 19
10 13
4 24
41 29
6 36
24 38
21 42
41 33
38 51
27 8
18 28
49 53
9 13
42 4
34 37
22 14
48 36
11 3
4 54
44 27
42 42
31 37
8 22
20 29
45 8
32 45
43 4
54 26
17 38
35 22
16 32
54 20
31 39
14 6
22 11
25 45
3 41
4 3
27 51
12 54
29 11
28 45
43 9
3 7
42 30
54 37
23 51
47 34
24 5
26 51
49 48
48 42
34 6
24 29
55 41
20 3
32 6
15 49
40 22
4 1
55 54
36 19
2 4
29 34
51 24
47 25
53 40
1 10
23 24
22 30
25 38
6 51
20 42
17 1
14 23
11 32
6 21
9 15
5 16
27 55
6 22
39 18
42 25
9 36
2 33
15 42
1 12
14 42
22 51
47 48
52 7
31 40
33 30
34 47
9 33
28 41
54 39
39 16
5 24
23 7
45 5
30 35
2 50
38 6
49 20
36 38
8 16
30 42
48 3
11 43
29 15
24 25
42 10
43 26
7 52
38 5
25 12
26 52
54 31
22 31
44 53
33 7
10 48
6 31
21 3
37 28
51 17
5 38
36 26
29 36
33 52
16 29
35 49
5 33
45 27
54 54
7 53
21 29
51 15
8 18
26 44
34 16
31 12
39 32
20 7
37 9
37 33
26 1
29 9
33 38
46 28
2 6
7 38
54 7
55 40
54 5
51 20
34 1
37 4
6 45
25 51
49 33
5 25
30 14
52 20
29 47
22 47
36 35
51 26
9 18
6 29
29 7
40 34
27 16
13 18
37 37
32 21
11 45
24 53
49 5
39 44
21 16
17 10
41 55
2 30
40 52
12 11
20 38
25 48
24 15
47 20
13 43
13 10
17 32
21 50
52 54
25 23
41 15
15 36
21 7
31 1
42 3
37 21
20 21
53 11
35 39
38 14
18 17
14 31
1 26
11 19
3 38
13 11
5 45
30 48
52 49
7 32
38 24
38 18
18 22
39 48
16 51
6 48
17 33
42 1
8 17
30 18
55 49
45 38
43 11
4 47
33 20
38 50
44 25
9 21
29 24
49 37
43 17
13 27
18 33
39 13
19 36
25 27
