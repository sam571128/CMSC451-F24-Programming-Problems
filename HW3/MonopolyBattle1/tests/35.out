2044 2044 2044 2044 2044 2044 2044 2044 2044 2541 2044 2044 2044 2082 2044 2044 2044 2044 2044 2044 2044 2044 2209 2044 2044 2044 2044 2044 2044 2044 2044 2044 2044 2044 2285 2044 2044 2044 2044 2876 2044 2044 2044 2406 2728 2044 2044 2044 2044 2044 2044 2044 2044 2044 2044 